src/vstest.console/vstest.console.csproj

```diff
@@ -13,10 +13,11 @@
     <PlatformTarget Condition="'$(TargetFramework)' == 'net451'">AnyCPU</PlatformTarget>
     <Prefer32Bit Condition="'$(TargetFramework)' == 'net451'">true</Prefer32Bit>
     <ApplicationManifest>app.manifest</ApplicationManifest>
+    <RootNamespace>Microsoft.VisualStudio.TestPlatform.CommandLine</RootNamespace>
   </PropertyGroup>
   <PropertyGroup Condition="'$(TargetFramework)' != 'netcoreapp2.1'">
     <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
-  </PropertyGroup>
+  </PropertyGroup> 
```

Comment:
Ah now I see why the resources, are fixed.

nit: Extra space?

Reply:
I am not sure why it's there. Removed.